gymnasium/envs/mujoco/half_cheetah_v5.py

```diff
@@ -136,6 +138,7 @@ class HalfCheetahEnv(MujocoEnv, utils.EzPickle):
         - Added `frame_skip` argument, used to configure the `dt` (duration of `step()`), default varies by environment check environment documentation pages.
         - Restored the `xml_file` argument (was removed in `v4`).
         - Renamed `info["reward_run"]` to `info["reward_forward"]` to be consistent with the other environments.
+        - Fixed the observation space in doc string to be consistent with the xml file.
```

Comment:
This does not belong here, remove it. Documentation changes are listed in the gymnasium release notes